src/NServiceBus.Testing.Tests/Saga/SagaTimeoutTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿namespace NServiceBus.Testing.Tests.Saga
+{
+    using System;
+    using System.Threading.Tasks;
+    using NUnit.Framework;
+    // ShouldPassExpectNotPublishWhenPublishingAndCheckingPredicateThatFails
+    [TestFixture]
+    class SagaTimeoutTests
+    {
+        [Test]
+        public void ShouldPassExpectSendWhenInvokingAllTimeoutCalls()
+        {
+            Test.Saga<TimeoutSaga>()
+                .WhenHandling<TimeoutSaga.StartMessage>()
+                .ExpectSend<TimeoutSaga.SendMessage1>()
+                .ExpectSend<TimeoutSaga.SendMessage2>()
+                .WhenSagaTimesOut();
+        }
+
+        [Test]
+        public void ShouldPassOneExpectSendWhenInvokingAllTimeoutsAfterATime()
```

Comment:
could that be something like `ShouldOnlyInvokeTimeoutsWithinSpecifiedTimeRange`?